src/content/docs/en/guides/rss.mdx

```diff
@@ -77,7 +80,8 @@ The RSS feed standard format includes the following values for each published it
 - `link`: A URL to the original source of the entry. (optional)
 - `pubDate`: The date of publication of the entry. (optional)
 - `content`: The full content of your post. (optional)
-- `customData`: A field for including any extra data, such as other frontmatter properties from your blog posts.
+
+<ReadMore>See [`item` configuration reference](https://github.com/withastro/astro/tree/main/packages/astro-rss#items) for a complete list of options.</ReadMore>
```

Comment:
I believe the option is actually `items`? I am not sure if you meant the configuration reference for an *individual* item, if so, then it should not contain backticks. Also, nit: missing "the".
```suggestion
<ReadMore>See the [`items` configuration reference](https://github.com/withastro/astro/tree/main/packages/astro-rss#items) for a complete list of options.</ReadMore>
```

Reply:
I believe Yan is correct here, it sounds like the intention is to point to the reference property!